src/functionBased/components/Header.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+const Header = () => {
+  const headerStyle = {
+    padding: '20px 0',
+    lineHeight: '1.5em',
+  };
+  return (
+    <header style={headerStyle}>
+      <h1
+        style={{
+          fontSize: '6rem',
+          fontWeight: '600',
+          marginBottom: '2rem',
+          lineHeight: '1em',
+          color: '#ececec',
+          textTransform: 'lowercase',
+          textAlign: 'center',
+        }}
```

Comment:
Kindly keep the styling in the CSS file since it is not in condition. By this, the file will be more readable.